src/xx-info

```diff
@@ -147,6 +160,18 @@ if [ -z "$TARGETARCH" ]; then
     "x86_64")
       TARGETARCH="amd64"
       ;;
+    "x86_64-v2")
```

Comment:
How are these values possible?

Reply:
Didn't know what I was thinking about but was looking at LLVM `march` handling with `-march=x86-64-v[234]`: https://github.com/llvm/llvm-project/commit/012dd42e027e2ff3d183cc9dcf27004cf9711720#diff-ec770381d76c859f5f572db789175fe44410a72608f58ad5dbb14335ba56eb97R193

I guess we should check cpu features to handle that?

Reply:
Don't think this should be implied from current system at all. Building in v4 system does not mean that the default build target should be v4. It should still be v1 for max compatibility.

Reply:
> It should still be v1 for max compatibility.

Good point